.github/workflows/build.yml

```diff
@@ -53,8 +54,9 @@ jobs:
           python-version: 3.8
       - name: Store installed Python version
```

Comment:
Don't we also want to update the `python-version` here to `3.9` as well?

Reply:
Yes.  See commit 2ca9b0a.